load/UpdateDBPFilesetTables.py

```diff
@@ -294,10 +294,10 @@ def getDuration(self, file):
 	from DBPLoadController import *
 
 	config = Config.shared()
-	lptsReader = LPTSExtractReader(config.filename_lpts_xml)
-	filesets = InputFileset.filesetCommandLineParser(config, lptsReader)
+	languageReader = LanguageReaderCreator().create(config)
+	filesets = InputFileset.filesetCommandLineParser(config, languageReader)
```

Comment:
@bradflood , I question by curious. I was checking the class InputFileset and I didn't see the method: filesetCommandLineParser. I tried to run locally this file I got the next error message:


```shell
AttributeError: type object 'InputFileset' has no attribute 'filesetCommandLineParser'
```

Reply:
I've updated it with this:
languageReader = LanguageReaderCreator().create(config)
	filesets = InputProcessor.commandLineProcessor(config, AWSSession.shared().s3Client, languageReader)